client/jetpack-connect/controller.js

```diff
@@ -31,6 +34,12 @@ import { isJetpackSite } from 'state/sites/selectors';
  */
 const debug = new Debug( 'calypso:jetpack-connect:controller' );
 const userModule = userFactory();
+const analyticsPageTitleByType = {
+	install: 'Jetpack Install',
+	personal: 'Jetpack Connect Personal',
+	premium: 'Jetpack Connect Premium',
+	pro: 'Jetpack Install Pro',
```

Comment:
Yup, this is probably my fault, I'm sure I messed it up and write install instead of connect. But @seear has a point, it makes sense to keep it like that to keep being consistent with the old results. Also, this has been around for a year or so and nobody in growth have complained, so I guess it's ok :D